net.crossfireatlas.crossfire.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: net.crossfireatlas.crossfire
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
```

Comment:
I assume it isn't meant to be empty.

Reply:
```
(flatpak-builder:2): Json-WARNING **: 11:11:30.531: Failed to deserialize "sdk-extensions" property of type "GStrv" for an object of type "BuilderManifest"
```

Reply:
I have added: org.freedesktop.Sdk.Extension.openjdk11